cogs/tags.py

```diff
@@ -21,6 +22,35 @@ def build_tag_name(record):
         name += f" ({record.get('alias')})"
     return name
 
+async def convert_slash(self, inter, tag_name):
```

Comment:
why is there a function taking `self` outside of a class? did you mean to make it a method of `class Tags`?